dpdispatcher/ssh_context.py

```diff
@@ -115,7 +117,7 @@ def _setup_ssh(self):
         self.ssh.connect(hostname=self.hostname, port=self.port,
                         username=self.username, password=self.password,
                         key_filename=self.key_filename, timeout=self.timeout,passphrase=self.passphrase,
-                        compress=True,
+                        compress=self.tar_compress,
```

Comment:
This option is used for OpenSSH compression but not "tar". See also:
- https://man.openbsd.org/ssh#C
- https://man.openbsd.org/ssh_config.5#Compression